### ISSUE TITLE
questions about hw2 

### ISSUE THREAD

**Issue by rkdgnlgus2** [swsnu/swppfall2021#21]
Hello, I have 2 questions about hw2.
Can we assume that babyname.report.csv file always has header(year, rank...) as a first row?
And can alert message have trailing newline?
(example : 
You must correct: 

First Name
Last Name
Email
Date of Birth(\n) << should I remove it?
)
Thank you.

**Comment by kdh0102** [comment]
> Can we assume that babyname.report.csv file always has header(year, rank...) as a first row?

Yes. The assumption is written in the function(`parseAndSave`) comment.

> And can alert message have trailing newline?

It actually does not matter in this case, but I suggest removing the newline at the end.
Note that any typo that does not match the requirements can lead to a minor point deduction.

**Comment by rkdgnlgus2** [comment]
Thank you!
